doc/user_guide/dynamo0p3.rst

```diff
@@ -2593,143 +2624,95 @@ as a ``real``-valued field are denoted with the keyword **plus**.
 X_plus_Y
 ^^^^^^^^
 
-**X_plus_Y** (*field3*, *field1*, *field2*)
+**X_plus_Y** (**field3**, *field1*, *field2*)
 
 Sums two fields and stores the result in the third field (``Z = X + Y``)::
 
   field3(:) = field1(:) + field2(:)
 
```

Comment:
Thanks for doing this. I think this means the comment referring to #1149 in dynamo0p3.rst can be removed.

Reply:
I realised this needed some rephrasing at the beginning of the Built-ins section. I extended the note that refers #1149 to include an "example" of the generated code and clarified that we use array notation for clarity.